cogs/System.py

```diff
@@ -0,0 +1,36 @@
+import discord
+from discord.ext import commands
+
+
+class System(commands.Cog):
+    def __init__(self, perceus):
+        self.perceus = perceus
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Logged in as: ')
+        print(self.perceus.user.name)
+        print(self.perceus.user.id)
+        print('--------------')
```

Comment:
I recommend putting on_ready event in main file instead of a file that is hidden in cogs folder.